src/Geosuggest.jsx

```diff
@@ -399,6 +399,8 @@ class Geosuggest extends React.Component {
       suggestionsList = <SuggestList isHidden={this.state.isSuggestsHidden}
         style={this.props.style.suggests}
         suggestItemStyle={this.props.style.suggestItem}
+        suggestsClassName={this.props.suggestsClassName}
+        suggestItemsClassName={this.props.suggestItemsClassName}
```

Comment:
I slightly lean towards naming the props after the helper components. It's easier to understand what `suggest-list / suggest-item` are supposed to do than `suggests / suggest-items`. 
```js
suggestListClassName={this.props.suggestListClassName}
suggestItemClassName={this.props.suggestItemClassName}
```